docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,293 @@
+* **integrity**: Yes (partial? key registration process is transient and limited in time) 
+* **availability**: Yes (need access to aggregator to register key) 
+
+#### Mithril signatures diffusion
+
+Mithril signers produces signatures every time a new immutable file is created in the Chain DB. Those signatures are generated from a random lottery based on the signer's stake and the protocol parameters.
+Preventing mithril signers from signing decreases the number of signatures and could allow attacker to take control of the produced snapshot
+
+* **confidentiality**: No 
+* **availability**: Yes
+* **integrity**: No (signatures are considered tamper-proof)
+
+#### Mithril protocol parameters 
+
+Protocol parameters are needed to coordinate the production of valid multi-signatures. They are served by the aggregator
+
+* **confidentiality**: No (they actually need to be public)
+* **availability**: Yes (partial?)
+* **integrity**: Yes (tampering protocol parameters can lead a signer to produce invalid signatures)
+
+#### Era reader configuration files
+
+The era reader [Era address](https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/release-mainnet/era.addr) and [Era vkey](https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-infra/configuration/release-mainnet/era.vkey) are used by signers to extract information about the current Mithril Era which defines the structure of snapshots and therefore signatures
+
+#### Era reader signing key
+
+The key tied to the aforementioned era address 
+
+### Client-side only assets
+
+These are the assets that are relevant only when downloading and verifying certificates and full-node snapshots. 
+
+:::info To do
+
+Verification process being trustless, does it really make sense to include those in the Threat Model?
+
+:::
+
+#### Mithril certificate verification process
+
+Mithril clients download snapshots and verify associated certificates using the mithril-client library, either from a CLI tool or [embedded in a browser](https://mithril.network/explorer/) 
+
+* **availability**: Yes (?)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril snapshots & certificates
+
+Mithril certificates are produced by the aggregator from the individual signatures. Mithril certificates' security rests upon a chain of trust ultimately pointing at the Genesis certificate. Mithril aggregator maintains snapshots and certificates (aggregated multi-signatures) to be served to clients
+
+* **availability**: No (Snapshots and certificates are just a fallback, a cardano-node client can always retrieve the infromation from the cardano network itself, albeit much more slowly)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
```

Comment:
Maybe this could be limited to certificates only?

Reply:
A separate section for the snapshot (or more general the artifacts ?) could make sense too.

Reply:
I added a new section for "Mithril artifacts" :+1: